usecases/schema/update.go

```diff
@@ -266,25 +267,101 @@ func (m *Manager) CreateSnapshot(ctx context.Context, principal *models.Principa
 	}
 }
 
+func (m *Manager) RestoreSnapshotStatus(ctx context.Context, principal *models.Principal,
+	className, storageName, ID string,
+) (string, string, string, error) {
```

Comment:
It would maybe help the code to be a bit more readable if we return a struct here, or perhaps to name the return arguments here in the function signature

Reply:
Yeah, I was about to suggest the same